Specs/Scene/ClippingPlaneSpec.js

```diff
@@ -1,5 +1,5 @@
 defineSuite([
-        'Core/ClippingPlane',
+        'Scene/ClippingPlane',
```

Comment:
Here too.

Reply:
Whoops, to be fair I thought the define for the class being tested had to be first, but I hadn't looked at, say, the spec for `Scene`...

Reply:
@likangning93 you are correct, the define for the class being tested does need to be first (because of Cesium specifics).

Reply:
Yup sorry, I overlooked this case.